lib/ts/runroot.h

```diff
@@ -0,0 +1,35 @@
+/** @file
+
+  A brief file prefix
+
+  @section license License
+
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+// runroot handler for TS_RUNROOT
+// detailed information in runroot.cc
+
+#pragma once
+
+#include <string>
+
+#define MAX_CWD_LEN 1024
+
+std::string check_parent_path(const std::string &path, bool json = false);
```

Comment:
Hmmm, how is this passing clang-format?  :)

Reply:
I thought there was a file that gets updated to work on certain files. I had a similar issue with a different PR.

Reply:
@zwoop this is because it is a function declaration not a definitions. From what I see declarations are not being change to the 

return_val
func_name( args )

form